uisdk/src/main/java/com/karhoo/uisdk/screen/booking/booking/payment/PaymentMVP.kt

```diff
@@ -8,19 +8,21 @@ interface PaymentMVP {
 
     interface View {
 
-        fun bindCardDetails(savedPaymentInfo: SavedPaymentInfo?)
+        fun bindPaymentDetails(savedPaymentInfo: SavedPaymentInfo?, quotePrice: QuotePrice? = null)
 
         fun handlePaymentDetailsUpdate(braintreeSDKNonce: String?)
 
-        fun initialisePaymentFlow(price: QuotePrice?)
+        fun initialiseChangeCard(price: QuotePrice?)
 
         fun initialiseGuestPayment(price: QuotePrice?)
 
+        fun initialisePaymentFlow(price: QuotePrice?)
+
         fun showError(@StringRes error: Int)
 
         fun showPaymentDialog(braintreeSDKToken: String)
 
-        fun showPaymentUI(braintreeSDKToken: String)
+        fun showPaymentUI(sdkToken: String, paymentData: String? = null, price: QuotePrice?)
```

Comment:
Since paymentData is an optional variable, I guess we dont need to assign null to it?
`paymentData: String? = null`

Reply:
Seems that we need the default here or we need to pass the value in